apps/client/src/app/components/paywall/StiggPaywall.tsx

```diff
@@ -68,6 +68,7 @@ export function Paywall({
         highlightedPlanId="plan-todos-essentials"
         onPlanSelected={onSubscribe}
         textOverrides={textOverrides}
+        billingCountryCode="JP"
```

Comment:
WDYT to use [node-geoip](https://github.com/geoip-lite/node-geoip) to show different paywalls based on the user location?
This would be super cool way to demo it 🚀

Reply:
+1 +1 +1